client/src/components/ElementEditor/ElementList.js

```diff
@@ -154,9 +161,7 @@ class ElementList extends Component {
     }
 
     let output = blocks.map(element => {
-      const saveElement = this.state.saveAllElements
-        && this.state.hasUnsavedChangesBlockIDs[element.id]
-        && this.state.validBlockIDs[element.id] === null;
+      const saveElement = this.state.saveAllElements && this.state.hasUnsavedChangesBlockIDs[element.id];
```

Comment:
We should attempt to save the block regardless of the validation state it was in prior to trying to save the block.